src/lib.rs

```diff
@@ -1503,3 +1516,7 @@ impl<T: Counter, F: Counter> PartialEq<Histogram<F>> for Histogram<T>
 #[path = "tests/tests.rs"]
 #[cfg(test)]
 mod tests;
+
+/// Serialization/deserialization support.
+#[path = "serialization/serialization.rs"]
```

Comment:
Yeah, I think the thing that bugs me the most is seeing a list of tabs:

`[mod.rs] [mod.rs] [mod.rs] [mod.rs]`

That and Cargo dependency handling are the places where I feel like the Rust community is backwards. :(